packages/ui/src/index.ts

```diff
@@ -28,6 +28,10 @@ export { Tag } from './components/Tag'
 export type { TagProps } from './components/Tag'
 
 export { Typography } from './components/Typography'
+
+export { List } from './components/List'
```

Comment:
Please place it in Alphabetical order, it's a common practice for an entry files...

Reply:
That's not consistent with rule of import order in component file...
In component we should order imports another way.
Can it be done with combination of eslint rules?

Reply:
I guess index.ts will be autogenerated later...but for now it's done by that to be easy to orient...

You can Ctrl+Click on lib to jump to it's type declarations, so UI-kit have only 1 big entry file...That's why it's important to be able to scan it for needed component easily...And that's why component-level imports is not that important (at least for now)